pkg/features/kube_features.go

```diff
@@ -595,6 +595,12 @@ const (
 	// Disable any functionality in kube-apiserver, kube-controller-manager and kubelet related to the `--cloud-provider` component flag.
 	DisableCloudProviders featuregate.Feature = "DisableCloudProviders"
 
+	// owner: @andrewsykim
+	// alpha: v1.22
```

Comment:
NIT: just see this: it would be 1.23

Reply:
https://github.com/kubernetes/kubernetes/pull/104278